pkg/reconciler/kubernetes/tektonconfig/controller.go

```diff
@@ -88,3 +93,13 @@ func NewExtendedController(generator common.ExtensionGenerator) injection.Contro
 		return impl
 	}
 }
+
+// customEnq adds `config` in work queue so that whenever a namespace
+// event occurs, the reconciler get triggered. The reconciler will only
+// get triggered only when a TektonConfig instance with name `config`
+// already exist.
+func customEnq(impl *controller.Impl) func(obj interface{}) {
```

Comment:
change this name to something descriptive, something like `enqueueName(impl *controller.Impl, name string)`.
so that we can pass the `common.ConfigResourceName`